lib/ansible/module_utils/nxos.py

```diff
@@ -116,9 +116,11 @@ def exec_command(self, command):
             command = self._module.jsonify(command)
         return exec_command(self._module, command)
 
-    def get_config(self, flags=[]):
+    def get_config(self, flags=None):
         """Retrieves the current config from the device or cache
         """
+        flags = flags = []
```

Comment:
This one needs manual fixup.

Reply:
Good catch. That one was wrong in the original commit too.